pkg/gui/controllers/helpers/merge_and_rebase_helper.go

```diff
@@ -51,17 +52,14 @@ func (self *MergeAndRebaseHelper) CreateRebaseOptionsMenu() error {
 		options = append(options, REBASE_OPTION_SKIP)
 	}
 
-	menuItems := make([]*types.MenuItem, len(options))
-	for i, option := range options {
-		// note to self. Never, EVER, close over loop variables in a function
```

Comment:
one benefit of using .Map is that you can't shoot yourself in the foot when closing over loop variables